Adds changes to lp rewards, price range details (#443)
vegaprotocol/documentation: docs/concepts/liquidity/provision.md

The Vega protocol allows liquidity to be priced individually for each market, a design decision that rewards liquidity providers for taking a bigger risk on markets with little liquidity competition by allowing them to earn more per trade, and drives down fees on markets where there are many participants committing liquidity.

> Providing liquidity on a market can be done using a combination of two tactics: [batched limit orders](./../trading-on-vega/orders#batch-order) and automatic orders created from a [liquidity commitment](#liquidity-commitments). Supplying limit orders makes an LP eligible to receive maker fees when volume they place on the book is hit, while a liquidity commitment makes an LP eligible to receive a portion of the liquidity fee from every trade on the book, regardless of if it their volume that was targeted or not. Using a liquidity provision strategy that combines both tactics means the provider is eligible to receive both fees.

Author: I thought that wasn't the case, but you would know, so I've amended your suggestion to cover that.

> Providing liquidity on a market can be done using a combination of two tactics: [batched limit orders](./../trading-on-vega/orders#batch-order) and automatic orders created from a [liquidity commitment](#liquidity-commitments). Supplying limit orders makes an LP eligible to receive maker fees when volume they place on the book is hit. A liquidity commitment makes an LP eligible to receive a portion of the liquidity fee from every trade on the book, regardless of if their volume that was targeted or not, and orders that are hit will trigger the maker fee. Using a liquidity provision strategy that combines both tactics means the provider is eligible to receive both fees.

Reviewer: ```suggestion
Providing liquidity on a market can be done using a combination of two tactics: [batched limit orders](./../trading-on-vega/orders#batch-order) and automatic orders created from a [liquidity commitment](#liquidity-commitments). Supplying limit orders makes an LP eligible to receive maker fees when volume they place on the book is hit. A liquidity commitment makes an LP eligible to receive a portion of the liquidity fee from every trade in the market and orders that are hit will earn the maker fee.
```

:::

### Orders created from commitment

> In essence, liquidity commitment orders are sets of pegged orders grouped by order book size, with a proportion set for each order within the order 'shape'. The overall volume that Vega will imply and automatically place on the order book from this depends on the remaining liquidity obligation, the best bid / ask prices, and the LP price range set on the market. The reference price, offsets and proportion at the reference / offset can always be amended. If an order created from the commitment ends up with a price outside the [price range set in the market proposal](#price-range-for-liquidity-orders), it's adjusted so it falls on the boundary. 

Reviewer: ```suggestion
In essence, liquidity commitment orders are sets of pegged orders grouped by order book side, with a proportion set for each order within the order 'shape'. The overall volume that Vega will imply and automatically place on the order book from this depends on the remaining liquidity obligation, the best bid / ask prices, and the LP price range set on the market. The reference price, offsets and proportion at the reference / offset can always be amended. If an order created from the commitment ends up with a price outside the [price range set in the market proposal](#price-range-for-liquidity-orders), it's adjusted so it falls on the boundary. 
```
